x509util/x509util.go

```diff
@@ -861,6 +861,34 @@ func ExtractSCT(sctData *x509.SerializedSCT) (*ct.SignedCertificateTimestamp, er
 	return &sct, nil
 }
 
+// ParseSCTListIntoSCTs serializes SCTs into SCT list.
```

Comment:
"Parse" implies deserialization. See ParseSCTsFromSCTList.

Reply:
Renamed to Marshal...